controllers/admin/AdminCmsController.php

```diff
@@ -272,7 +271,7 @@ public function displayList($token = null)
         /* Display list header (filtering, pagination and column names) */
         $this->displayListHeader($token);
         if (!count($this->_list)) {
-            echo '<tr><td class="center" colspan="'.(count($this->fields_list) + 2).'">'.$this->l('No items found').'</td></tr>';
+            echo '<tr><td class="center" colspan="'.(count($this->fields_list) + 2).'">'.$this->trans('No items found', array(), 'Admin.Global').'</td></tr>';
```

Comment:
I think it's the only occurence, so it can be Admin.Design.Notification for instance.